src/util/classes.ts

```diff
@@ -40,14 +43,29 @@ export const defaultCategories: Category[] = [
     data: { color: '#0FF' },
   },
   { name: ['Media', 'Video'], rule: { type: 'regex', regex: 'YouTube|Plex|VLC' } },
+  { name: ['Media', 'Image'], rule: { type: 'regex', regex: 'Gimp|Inkscape' } },
```

Comment:
Wouldn't image editing be a work-related thing?

Reply:
Well those tools are for editing so in this way is more specific and can be extended the video part also with video editors.
PS: I am italian and I like the nickname :-D

Reply:
That makes sense! @ErikBjare what do you think here? I have a couple changes along these lines I'd like to merge in.

My take on it is we should be more prescriptive about the initial categorization. Users can always customize it if they'd like to, but getting to a good set of defaults creates a better UX. In this case, I think most of the time if someone is editing images, they are doing it in a work-related way, so 'work > media' seems to make sense to me.

> PS: I am italian and I like the nickname :-D

Haha! Thanks :) I wish I was more authentically Italian, great to meet another italian dev :)

Reply:
@iloveitaly I made a comment in #174 with my thoughts, but I'm open to merging PRs like this if they cater to a broad audience :)

Although I do think the 'Media' category is for media *consumption* and not for media creation, that was my intent anyway :)

Reply:
Maybe a solution can be Work/Image Editing and Work/Video Editing and remove that Media/Image.
If it is fine for you I will updated the PR.

Reply:
I think that's a great idea!